dask/dataframe/io/tests/test_io.py

```diff
@@ -310,12 +310,14 @@ def test_from_pandas_single_row():
     assert_eq(ddf, df)
 
 
+@pytest.mark.skipif(np.__version__ < '1.11',
+                    reason='datetime unit unsupported in NumPy < 1.11')
```

Comment:
Really? I'm a bit surprised by this. IIRC datetime64 stabilized around 1.8 or 1.9?